Purchases/Networking/RCHTTPClient.m

```diff
@@ -97,6 +117,16 @@ - (void)performRequest:(NSString *)httpMethod
         }
     }
 
+    RetryRequestBlock retryBlock = ^void() {
+        RCDebugLog(RCStrings.network.retrying_request, httpMethod, path);
+        [self performRequest:httpMethod
+                    serially:performSerially
+                        path:path
+                        body:requestBody
+                     headers:headers
+                     retried:YES
+           completionHandler:completionHandler];
+    };
```

Comment:
I've updated this to remove the retryBlock and simply adding to the queue. The only thing I am not very convinced about is that I had to handleResponse to accept the `RCHTTPResponse`, otherwise I didn't have access to the original headers, path, etc. Let me know what you think